CONTRIBUTING.md

```diff
@@ -1129,7 +1129,7 @@ can be found [on the wiki][wiki-CUDW].
 [plugin-tutorial]: doc/plugin_tutorial
 [ProofGeneral-issues]: https://github.com/ProofGeneral/PG/issues
 [Reddit]: https://www.reddit.com/r/Coq/
-[refman]: https://coq.inria.fr/refman
+[refman]: https://coq.inria.fr/distrib/current/refman
```

Comment:
Actually both the former and the newer links are broken (hopefully it won't stay like that for too long though). The correct fix is to add a trailing `/`.

Reply:
Thank you for pointing it out, I was sure I checked the link before raising the PR.